java/src/org/openqa/selenium/remote/RemoteWebDriverBuilder.java

```diff
@@ -135,6 +135,10 @@ public class RemoteWebDriverBuilder {
     // Access through RemoteWebDriver.builder
   }
 
+  RemoteWebDriverBuilder(ClientConfig clientConfig) {
```

Comment:
I think we can revert this as well because we already have `RemoteWebDriver.builder().config(ClientConfig.defaultConfig())`